addons/mail/static/src/core/common/im_status.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ImStatus">
-        <div class="o-mail-ImStatus d-flex justify-content-center flex-shrink-0 align-items-center bg-white bg-opacity-100" t-attf-class="{{ props.className }}" t-att-style="props.style" t-att-class="{
+        <div class="o-mail-ImStatus d-flex justify-content-center flex-shrink-0 align-items-center bg-white bg-opacity-100" t-if="!(persona?.im_status === 'im_partner' and env.embedLivechat)" t-attf-class="{{ props.className }}" t-att-style="props.style" t-att-class="{
```

Comment:
I understand that it's convenient to add the `t-if` in only once place. However, I don’t like the idea. If we call a component, it should always output something to the DOM. It feels strange to me to call the  `<ImStatus> ` component and have it decide to render nothing.